movie-browser/src/components/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const URL = 'https://ghibliapi.herokuapp.com/films';
+
+class MovieList extends Component {
+  state = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    axios
+      .get(URL)
+      .then(res => {
+        this.setState({
+          movies: res.data,
+        });
+      })
+      .catch(err => {
+        console.log(err); // log the error?
+      });
+  }
+
+  render() {
+    const { movies } = this.state;
+    console.log('movies', movies);
```

Comment:
remove this `console.log`

Reply:
removed the line of code and will make sure to remove console.log in future commits or use React Dev Tools